src/CspNijmegenStack.ts

```diff
@@ -80,6 +81,13 @@ export class CspNijmegenStack extends cdk.Stack {
         }),
       },
     });
+
+    NagSuppressions.addResourceSuppressions(role, [{
+      id: 'AwsSolutions-IAM5',
+      reason: 'Aws gebruikt hier een wildcard in de originele PublicHostedZone class waar deze policy van geleend is',
```

Comment:
ik vermoed dat de * nog wel vervangen kan worden door bv een *.csp-nijmegen.nl, maar is niet echt een halszaak gezien het recht dat je hebt met de *